CHANGELOG.md

```diff
@@ -7,6 +7,9 @@ and this project adheres to [Semantic Versioning](http://semver.org/).
 
 ## Development
 
+### Changed
+- Way of getting available screen width and height
```

Comment:
Should we mention that we changed that to include screen density?

Reply:
Changed in https://github.com/bitmovin/bitmovin-analytics-collector-android/pull/118/commits/ccd9a3fcaab2a09c15624b04ec51bd0cab7e062d. Does this maybe has more sense?